### DISCUSSION TITLE
Awarded Framework Agreement

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#915] [Q&A] [answered]
Hello,

When a awarded framework agreement (<efbc:ContractFrameworkIndicator>true</efbc:ContractFrameworkIndicator>) is submitted, what should be submitted in the OPT-100-Contract node (NoticeDocumentReference node in the physical model)? The reference to the tender notice (notice type 16)? The reference to the last publication?

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/356bfe8d-6936-4fab-802f-30c4914ae3c9)

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
When it comes to a Contract within a Framework Agreement, it should be made reference to the Result Notice where these Framework Agreement has been awarded.

**Reply by AlexAS81** [reply]
I don't understand very well, we want to send the first awarded framework agreement of a notice (notice type 29), what other notice does it have to refer to?

**Reply by YvesJo** [reply]
Contract Framework Indicato (BT-768) is there to differentiate a Framework Contract (BT-768 = false) from a Contract within a Framework Agreement (BT-768 = true).
When the contract in the notice is about the Framework Agreement itself, then for that contract BT-768 has to be set to 'false' and OPT-100-Contract is not allowed.
When the contract is about a contract within a Framework agreement, then BT-768 has to be set to true and OPT-100-Contract shall refer to the result notice that contains the details about the framework agreement.
If both contracts (Framework Contract and first contract within that framework agreement) are published in the same notice, then there will be the need for a contract for each of them in that notice and the contract for the contract within the framework agreement should point to the notice itself (where the framework contract is to be found).
